include/irr/asset/material_compiler/IR.h

```diff
@@ -89,13 +108,15 @@ class IR : public core::IReferenceCounted
         for (INode* n : tmp)
             n->~INode();
         tmp.clear();
+        memMgr.freeLastAllocatedBytes(tmpSize);
+        tmpSize = 0u;
```

Comment:
make sure you don't trip up on padding that was inserted by alignment

Reply:
im sure